src/components/calcite-pick-list/calcite-pick-list.tsx

```diff
@@ -62,6 +62,11 @@ export class CalcitePickList {
    */
   @Prop({ reflect: true }) filterEnabled = false;
 
+  /**
+   * When true, the filter will stick to the top of the pick list when scrolling would otherwise hide filter.
+   */
+  @Prop({ reflect: true }) filterSticky = false;
```

Comment:
@pr3tori4n do we even need the option to disable this? It seems like we'd want it always to behave the same way.

Reply:
+1

Reply:
removed the prop.